common/test/acceptance/tests/discussion/helpers.py

```diff
@@ -76,22 +76,26 @@ def setup_cohort_config(self, course_fixture, auto_cohort_groups=None):
 
     def enable_cohorting(self, course_fixture):
```

Comment:
Remove always_divide_inline_discussions from this method so that other tests can include this mixin and not change the value always_divide_inline_discussions when it is not necessary for the tests.